### PR TITLE
*: change API document's update method

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
 change API document's update method, switch to swagger.
### What is changed and how it works?
- update the pull request template.
- remove old way

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Tests <!-- At least one of them must be included. -->
 Code changes
 
  - Has configuration change
- - Has HTTP API interfaces change (Don't forget to [update API document](https://github.com/pingcap/pd/blob/master/docs/development.md#updating-api-documentation))
+ - Has HTTP API interfaces change (Don't forget to [declarative comment API document](https://github.com/pingcap/pd/blob/master/docs/development.md#updating-api-documentation))
```
Comment (rleungx) [inline .github/pull_request_template.md:29]:
```suggestion
 - Has HTTP API interfaces change (Don't forget to [add the declarative comment for API](https://github.com/pingcap/pd/blob/master/docs/development.md#updating-api-documentation))
```

File: docs/development.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ This takes a while to run. The test suite uses a fork of [gocheck](http://labix.
 
 ## Updating API documentation
 
-We use [RAML 1.0](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md) to manage the API documentation, and the raml file is placed in `server/api/api.raml`. We also use [raml2html](https://github.com/raml2html/raml2html) to generate a more readable html file, which is placed in `docs/api.html`. When a PR involves API changes, you need to update the raml file within the same PR.
+We use [Swagger 2.0](https://swagger.io/specification/v2/) to automatically generate RESTful API documentation. When a PR involves API changes, you need to update the Go annotations, and the specific format can refer to [Declarative Comments Format](https://github.com/swaggo/swag#declarative-comments-format).
 
 Consider that raml2html depends on various npm packages and can only be run under a specific version of node. It is recommended to use docker to simplify the compilation of raml. You can update html file with [scripts/build-api.sh](../scripts/build-api.sh)
```
Comment (rleungx) [inline docs/development.md:37]: need to be removed?

**Comment by nolouch** [comment]
PTAL @rleungx @disksing

**Pull request merged**